dev/NavigationView/NavigationViewItemAutomationPeer.cpp

```diff
@@ -56,6 +56,20 @@ winrt::IInspectable NavigationViewItemAutomationPeer::GetPatternCore(winrt::Patt
     return result;
 }
 
+
+winrt::AutomationControlType NavigationViewItemAutomationPeer::GetAutomationControlTypeCore()
+{
+    // To be compliant with MAS 4.1.2, in DisplayMode 'Top',
+    //  a NavigationViewItem should report itsself as TabItem
+    if (IsOnTopNavigation()) {
```

Comment:
nit: the opening { should be on its own line.

Reply:
Fixed